views/confirmation/check-answers.pug

```diff
@@ -3,34 +3,46 @@ include ../_includes/address
 
 block variables
   -var title = __('Check your answers before filing')
+  -var isSenior = hasData(data, 'personal.dateOfBirth') && is65(data.personal.dateOfBirth)
+
 
 block content
 
   h1 #{title}
 
-  each section, index in answerInfo
-    div
-      .breakdown-table.breakdown-table--answers
-        h2.breakdown-table__heading #{__(index)}
-        dl.breakdown-table-data
-          each row in section
-            .breakdown-table__row.breakdown-table__row--answers
-              dt.breakdown-table__row-key #{__(row.text)}
-
-              if (row.text.toLowerCase().includes('address'))
-                dd.breakdown-table__row-value
-                  +address(data)
-              else
-
-                //- we don't want to translate the name, dollar amounts, or birthday (formatting the birthday is done prior to hitting the template in checkAnswersFormat)
-                if (row.text.toLowerCase().includes('name') || row.data.includes('$') || row.text.toLowerCase().includes('birth'))
-                  dd.breakdown-table__row-value #{row.data}
+  div.breakdown-table--check-answers
+    each section, index in answerInfo
+      div
+        .breakdown-table.breakdown-table--answers
+          h2.breakdown-table__heading #{__(index)}
+          dl.breakdown-table-data
+            each row in section
+              .breakdown-table__row.breakdown-table__row--answers
+                if (row.urlPath && row.urlPath.includes('confirm-income') && isSenior)
+                  dt.breakdown-table__row-key #{__(row.text.replace('$12,070', '$19,564').replace('12\u00A0070\u00A0$', '19\u00A0564\u00A0$'))}
```

Comment:
This is pretty clever 👍

Reply:
It's certainly not the nicest solution, but will do for now 😅